src/amber/cli/templates/scaffold/controller/granite/src/controllers/{{name}}_controller.cr.ecr

```diff
@@ -63,4 +63,26 @@ class <%= @name.capitalize %>Controller < ApplicationController
     end
     redirect_to "/<%= @name %>s"
   end
+
+  def <%= @name %>_params
+    params.validation do
+      <%- @fields.each do |field| -%>
+      <%- unless %(created_at updated_at).includes? field.name -%>
+      <%- if %w(Int32 Int64).includes? field.cr_type -%>
+      required(:<%= field.name %>) { |p| !JSON.parse(p.as(String)).as_i.nil? }
+      <%- elsif %w(Bool).includes? field.cr_type -%>
+      required(:<%= field.name %>) { |p| JSON.parse(p.as(String)).as_bool }
```

Comment:
Why is everything being parsed as JSON? Most params won't be JSON.

Reply:
I will be removing this `{ |p| !JSON.parse(p.as(String)).as_i.nil? }` to this `{ |p| !p.nil? }`